src/main/java/seedu/duke/project/Project.java

```diff
@@ -11,6 +12,7 @@
     protected boolean isDone;
     private TaskList tasks;
     private String projectDescription;
+    public static ArrayList<TeamMember> members = new ArrayList<>();
```

Comment:
Sam I think you got to make this private and make use of getters and setters, bcus i dont think it's good practice to make class members private.

Reply:
ok! thanks man